### DISCUSSION TITLE
how to set tileOptions of a layer

### DISCUSSION THREAD

**Discussion by scil** [ghettovoice/vuelayers#498] [Q&A] [answered]
I  want to make a tile layer `layer.tileOptions.crossOriginKeyword = null;`    
to resove [cors issue](https://gis.stackexchange.com/questions/71715/enabling-cors-in-openlayers)

how could i do ?

**Comment by ghettovoice** [comment, marked as answer]
Hello @scil ,
starting from the openlayers v3 cross-origin options was moved in to the source component.
So in vuelayers each tile image source component has a prop `crossOrigin`. You can set it to `anonymous` to avoid CORS issues with tile layers.

```html
// tiled WMS source
<vl-layer-tile>
  <vl-source-tile-wms cross-origin="anonymous" ... />
</vl-layer-tile>

// XYZ source
<vl-layer-tile>
  <vl-source-xyz cross-origin="anonymous" ... />
</vl-layer-tile>
```
